dashboard/app/controllers/courses_controller.rb

```diff
@@ -155,6 +155,14 @@ def code
 
   def standards
     unit_group = UnitGroup.get_from_cache(params[:course_name])
+    if !unit_group.present? && UnitGroup.family_names.include?(params[:course_name])
+      redirect_to_course = UnitGroup.all_courses.
+        select {|c| c.family_name == params[:course_name] && c.stable?}.
+        sort_by(&:version_year).
+        last
+      redirect_to "/courses/#{redirect_to_course.name}/standards"
```

Comment:
can we use a Rails url helper to construct this path, rather than hardcoding it?